integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/car/deployment/test/ESBJAVA4116CAppArtifactDeploymentFailureDueToNameTestCase.java

```diff
@@ -23,34 +23,20 @@
 import org.testng.annotations.Test;
```

Comment:
License should be updated with the created year

Reply:
It's already there :)